docs/semgrep-ci/ci-environment-variables.md

```diff
@@ -56,7 +56,20 @@ export SEMGREP_BASELINE_REF="main"
 
 ### `SEMGREP_BASELINE_COMMIT`
 
-Set `SEMGREP_BASELINE_COMMIT` to a commit hash to only show results that are **not** found in that hash. This environment variable doesn't work if you are not currently in a Git directory, there are unstaged changes, or given baseline hash doesn't exist.
+Set `SEMGREP_BASELINE_COMMIT` to a commit hash to only show results that are **not** found in that hash. Generally this is used to enable **[diff-aware scanning](/deployment/customize-ci-jobs#set-up-diff-aware-scans)** for CI providers that are **not** GitHub Actions or GitLab CI/CD. 
+
+This environment variable doesn't work if you are not currently in a Git directory, there are unstaged changes, or the given baseline hash doesn't exist or is not available in the CI environment.
+
+If you set `SEMGREP_BASELINE_COMMIT` in CI to enable diff-aware scanning, the ideal value is the git merge-base between the branch being scanned and the target branch that the code will be merged into. For example:
+
+```bash
+export SEMGREP_BASELINE_COMMIT=$(git merge-base main feature-brach)
+```
+To avoid harcoding the branch names, check your CI provider's documentation for available variables that provide the correct values for every CI job. For example, in a Jenkins environment:
```

Comment:
```suggestion
To avoid hardcoding the branch names, check your CI provider's documentation for available variables that provide the correct values for every CI job. For example, in a Jenkins environment, you can use:
```

Reply:
tx for the catch!